src/Commands/Env/ViewCommand.php

```diff
@@ -49,24 +53,8 @@ public function view($site_env, $options = ['print' => false,])
             return $url;
         }
 
-        // Otherwise attempt to launch it.
-        $cmd = '';
-        switch (php_uname('s')) {
-            case 'Linux':
-                $cmd = 'xdg-open';
-                break;
-            case 'Darwin':
-                $cmd = 'open';
-                break;
-            case 'Windows NT':
-                $cmd = 'start';
-                break;
-        }
-        if (!$cmd) {
-            throw new TerminusException("Terminus is unable to open a browser on this OS");
-        }
-        $command = sprintf('%s %s', $cmd, $url);
-        exec($command);
+        $this->getContainer()->get(LocalMachineHelper::class)->openUrl($url);
```

Comment:
Yeah. I was debating whether this helper rose to the level of deserving it's own inflector and `XAwareInterface/XAwareTrait` combo.

Right now we have 2 ways of handling dependencies: Automatic injection with reflectors and pulling directly from the container. We don't have good guidelines as to when one is appropriate and when the other is. Using `getHelper` adds a third. I'm not sure how I feel about that. Not necessarily agin it, but not necessarily fir it. I'll do some reading.